doc/rst/technotes/libraries.rst

```diff
@@ -269,6 +269,16 @@ requires the following:
 - ``Cython``
 - ``numpy``
 
+If you are on a system where libraries are built to be position dependent by
+default (e.g.  not OSx), you will need to set the environment variable
+``CHPL_LIB_PIC`` to ``pic`` and perform a ``make`` command from ``$CHPL_HOME``.
+This will cause the Chapel runtime and third-party libraries to be built with
+position independent code, which Python requires.  Note that position
```

Comment:
Can we change "which Python requires" to "which Python interoperability requires"?